PaulasWeatherApp/PaulasWeatherApp/ViewController/ViewController.swift

```diff
@@ -8,48 +8,40 @@
 import UIKit
 
 class ViewController: UITabBarController {
-    var countryCityName: String?
-    var currentWeather: WeatherDetails?
-    var hourlyWeatherList = [WeatherDetails]()
-    var dailyWeatherList = [Daily]()
+    
+    let WeatherVC = CurrentWeatherViewController()
+    let MetricsVC = WeatherReportsViewController()
+    let SettingsVC = SettingsViewController()
```

Comment:
property names will always start with small letter
+ no need to have them public